src/main/java/com/dataguise/saas/dto/SourceCountDTO.java

```diff
@@ -0,0 +1,23 @@
+package com.dataguise.saas.dto;
+
+public class SourceCountDTO {
+
+    private Integer cloud_sources;
+    private Integer on_premise_sources;
+
+    public Integer getCloud_sources() {
+        return cloud_sources;
+    }
+
+    public void setCloud_sources(Integer cloud_sources) {
```

Comment:
**Reporter**: PMD    
    **Rule**: FormalParameterNamingConventions
    **Severity**: ERROR
    **File**: /work/src/main/java/com/dataguise/saas/dto/SourceCountDTO.java L12
    
    The method parameter name &#39;cloud_sources&#39; doesn&#39;t match &#39;[a-z][a-zA-Z0-9]*&#39;&#10;&#10;&#10;Code Style https://pmd.github.io/pmd/pmd_rules_java_codestyle.html#formalparameternamingconventions
    
*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1892950829>*